### PR TITLE
Error removed and name added

### LINKED ISSUE #5: Get back daily meeting notifications
On Mondat—Friday send Daily Meeting notification at a specific time

### LINKED ISSUE #7: GraphqlError: You must provide a `first` or `last` value to properly paginate the `assignees` connection.
```
GraphqlError: You must provide a `first` or `last` value to properly paginate the `assignees` connection.
```

```
app_1  |   errors: [
app_1  |     {
app_1  |       type: 'MISSING_PAGINATION_BOUNDARIES',
app_1  |       path: [ 'node', 'cards', 'nodes', 2, 'content', 'assignees' ],
app_1  |       locations: [ { line: 29, column: 15 } ],
app_1  |       message: 'You must provide a `first` or `last` value to properly paginate the `assignees` connection.'
app_1  |     },
app_1  |     {
app_1  |       type: 'MISSING_PAGINATION_BOUNDARIES',
app_1  |       path: [ 'node', 'cards', 'nodes', 3, 'content', 'assignees' ],
app_1  |       locations: [ { line: 29, column: 15 } ],
app_1  |       message: 'You must provide a `first` or `last` value to properly paginate the `assignees` connection.'
app_1  |     }
app_1  |   ],
```

```
app_1  |   request: {
app_1  |     query: '\n' +
app_1  |       'query($id: ID!){\n' +
app_1  |       '  node(id: $id) {\n' +
app_1  |       '    ... on ProjectColumn {\n' +
app_1  |       '      name\n' +
app_1  |       '      cards(first: 30) {\n' +
app_1  |       '        __typename\n' +
app_1  |       '        totalCount\n' +
app_1  |       '        nodes {\n' +
app_1  |       '          id\n' +
app_1  |       '          note\n' +
app_1  |       '          state\n' +
app_1  |       '          creator {\n' +
app_1  |       '            login\n' +
app_1  |       '          }\n' +
app_1  |       '          content {\n' +
app_1  |       '            ... on PullRequest {\n' +
app_1  |       '              id\n' +
app_1  |       '              __typename\n' +
app_1  |       '              url\n' +
app_1  |       '              author {\n' +
app_1  |       '                login\n' +
app_1  |       '              }\n' +
app_1  |       '            }\n' +
app_1  |       '            ... on Issue{\n' +
app_1  |       '              id\n' +
app_1  |       '              url\n' +
app_1  |       '              __typename\n' +
app_1  |       '              assignees{\n' +
app_1  |       '                nodes{\n' +
app_1  |       '                  login\n' +
app_1  |       '                }\n' +
app_1  |       '              }\n' +
app_1  |       '            }\n' +
app_1  |       '          }\n' +
app_1  |       '        }\n' +
app_1  |       '      }\n' +
app_1  |       '    }\n' +
app_1  |       '  }\n' +
app_1  |       '}\n',
app_1  |     variables: { id: '...' }
app_1  |   }
```

### PR DESCRIPTION
**Pull request by robonetphy**
resolve #7 
resolve #5

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -12,7 +12,8 @@ COLUMN_NODE_ID=
 NOTIFIER_URL=
 
 # List of people to tag in the «Pull Requests» message
-MENTION = "f0m41h4u7 specc nikmel2803 talyguryn GeekaN n0str neSpecc"
+MENTION = "f0m41h4u7 specc nikmel2803 talyguryn GeekaN2 n0str neSpecc"
```
Comment (talyguryn) [inline .env.example]:
```suggestion
MENTION = "ilyamore88 fabled228 Xemka kebyo GeekaN2 ranemihir robonetphy neSpecc nikmel2803 talyguryn gohabereg khaydarov"
```

File: .env.example [hunk 2]
```diff
@@ -12,7 +12,8 @@ COLUMN_NODE_ID=
...
 
 # If using docker-compose set time there
 PR_TIME="0 9,18 * * 1-5"
```
Comment (talyguryn) [inline .env.example]:
```suggestion
PR_TIME="0 9,20 * * 1-5"
```

File: src/index.js [hunk 3]
```diff
@@ -13,7 +13,11 @@ const MENTION = process.env.MENTION;
  * At minute 0 past hour 9 and 18 on every day-of-week from Monday through Friday.
  */
 const PR_TIME = process.env.PR_TIME || '0 9,18 * * 1-5';
```
Comment (talyguryn) [inline src/index.js]:
```suggestion
const PR_TIME = process.env.PR_TIME || '0 9,20 * * 1-5';
```

File: src/index.js [hunk 4]
```diff
@@ -205,6 +209,14 @@ async function notifySprintsBacklogs() {
   return dataToSend;
 }
 
+/**
+ * Meeting Message to display
+ */
+const MEETING_MSG = `☝️ Join the meeting in Discord!
+@specc @guryn @khaydarovm @nikmel2803 @xemk4
```
Comment (talyguryn) [inline src/index.js]: it would be great if this list will be created automatically by MENTION list may be

File: src/index.js [hunk 5]
```diff
@@ -213,17 +225,29 @@ async function main() {
...
       notify(await notifySprintsBacklogs())
-        .then(() => console.log('Job completed'))
+        .then(() => console.log('PR Job Completed.'))
         .catch(console.error);
     },
     null,
     true,
     'Europe/Moscow'
   );
+  const meetingJob = new CronJob(MEETING_TIME, () => {
```
Comment (talyguryn) [inline src/index.js]:
```suggestion

  const meetingJob = new CronJob(MEETING_TIME, () => {
```

File: src/index.js [hunk 6]
```diff
@@ -213,17 +225,29 @@ async function main() {
...
+    notify(MEETING_MSG)
+      .then(() => console.log('Meeting Job Completed.'))
+      .catch(console.error);
+  },
+  null,
+  true,
+  'Europe/Moscow'
```
Comment (talyguryn) [inline src/index.js]: please add two spaces for lines 239-242

File: src/index.js [hunk 7]
```diff
@@ -213,17 +225,29 @@ async function main() {
...
+  );
 
   job.start();
   console.log('Notifier started');
-  console.log('Will notify at ' + PR_TIME);
+  console.log('Will notify at:' + PR_TIME);
+  meetingJob.start();
```
Comment (talyguryn) [inline src/index.js:262]:
```suggestion

  meetingJob.start();
```

**Review by talyguryn — CHANGES_REQUESTED** [review]

**Review by fabled228 — APPROVED** [review]

**Review by talyguryn — APPROVED** [review]

**Pull request merged**